### ISSUE TITLE
Mobile Image Card should display title outside of image

### ISSUE THREAD

**Issue by slowbot** [bloom-housing/bloom#2268]
**Describe the solution you'd like**
Mobile image card should have header above the image on browse and below the image on single listing for accessibility and legibility

**Describe alternatives you've considered**
Using the image gradient on mobile obscures the image.

**Additional context**
SF Dahlia has a solution for this we could roll into our component. 

**What is the acceptance criteria/definition of done?**
User should be able to see the title of the listing and the image clearly on mobile.
Gradient is removed on mobile

**Acceptance Criteria**
- On Desktop Browse page, Listing Name and Address should render to the right of the image, above the unit table 
- On Mobile Browse page, Listing Name and Address should render above the image
- On Single Listing, Listing Name and Address should render under the image
- Remove all caps and gradient overlap, implement as the default style with caps and gradient as optional props

**QA Review Instructions**

- Go to the browse or single listing page on desktop
- Observe the title overlay on the image
- Adjust the width of the screen to mobile
- You should see the listing name and address shown above the listing image on browse and below the listing image on single listing

**Updated Component Variants**
https://www.figma.com/file/tUtgsJO8XnF9Ij2dwu22EF/bloom-public-general?node-id=1005%3A3001

![image](https://user-images.githubusercontent.com/13915/153896496-eb37533c-137a-47ec-b146-404448f7cd56.png)

**Listings**
![image](https://user-images.githubusercontent.com/13915/144637307-fcaa1d12-5bce-4e59-884b-97b9735de0c1.png)

**Single**
![image](https://user-images.githubusercontent.com/13915/144928853-51b62f94-0c81-41fd-aa71-a1db8bf32d93.png)

**Comment by akegan** [comment]
@yindipei FYI that this issue is also relevant

**Comment by emilyjablonski** [comment]
QA Notes: Ensure the new cards on the Bloom dev environment match the designs on desktop and mobile.

**Comment by slowbot** [comment]
@emilyjablonski 

Line heights for card header should be 1.25
Mobile header, sub header, developer and view map link elements should be centered

On single listing, card header should be black

![image](https://user-images.githubusercontent.com/13915/162647585-88611fcf-40d6-4422-a32f-72bfacb2c7fd.png)

**Comment by emilyjablonski** [comment]
QA Notes: Ensure the ListingCard titles are centered mobile views, that the padding is adjusted as requested, and that the listing title is black on the detail pages.

**Comment by slowbot** [comment]
@emilyjablonski Looks good to me
